src/content/developers/tutorials/how-to-use-tellor-as-your-oracle/index.md

```diff
@@ -13,7 +13,7 @@ sourceUrl: https://docs.tellor.io/tellor/
 
 Pop Quiz: Your protocol is just about finished, but it needs an oracle to plug in to get an active off chain data feed going...What do you do?
 
-## (Soft) Prerequisites
+## (Soft) Prerequisites {#in-this-tutorial-youll-learn-how-to}
```

Comment:
Bummer - this script doesn't appear to be working correctly - the header id _should_ match the header text (i.e. this should be:
```
## (Soft) prerequisites {#soft-prerequisites}
```